frontend/apps/reader/modules/readersearch.lua

```diff
@@ -2,7 +2,9 @@ local BD = require("ui/bidi")
 local ButtonDialog = require("ui/widget/buttondialog")
 local InputContainer = require("ui/widget/container/inputcontainer")
 local InputDialog = require("ui/widget/inputdialog")
+local Notification = require("ui/widget/notification")
 local UIManager = require("ui/uimanager")
+local T = require("ffi/util").template
 local logger = require("logger")
 local _ = require("gettext")
 
```

Comment:
Close. ^_^ (See the style guide.)
```suggestion
local logger = require("logger")
local _ = require("gettext")
local T = require("ffi/util").template
```